### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# kochi
\ No newline at end of file
+# kochi
+Library adds the ability to log HTTP requests and responses
```
Comment (mingazhev) [inline README.md:2]: Which service is this client for?

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+                {
+                    var content = request.Content?.ReadAsStringAsync().GetAwaiter().GetResult();
+                    return !string.IsNullOrEmpty(content) && content.Contains("HotelBookingRequest");
+                };
+            });
+        ...
+    }
+}
+```
+Add settings to appsettings.json
```
Comment (mingazhev) [inline README.md:26]: what is this setting for?

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+```csharp
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        var fukuokaOptions = vaultClient.Get(Configuration["Fukuoka:Options"]).GetAwaiter().GetResult(); 
+        services.AddHttpClient("ClientName")
+            .AddHttpRequestLoggingHandler(options =>
+            {
+                options.Endpoint = fukuokaOptions["endpoint"];
```
Comment (mingazhev) [inline README.md:15]: Please add a comment for each setting

File: README.md (hunk 1 above)
Comment (DFSko) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (DFSko) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (DFSko) [reply README.md:24]: Fixed

**Review by mingazhev — APPROVED** [review]
Much better

**Pull request merged**